iot-app-kit.sh

```diff
@@ -0,0 +1,200 @@
+#!/usr/bin/env bash
+
+bold=$(tput bold)
+normal=$(tput sgr0)
+
+PACKAGE_LIST=(
+  "@iot-app-kit/core"
+  "@iot-app-kit/core-util"
+  "@iot-app-kit/dashboard"
+  "@iot-app-kit/react-components"
+  "@iot-app-kit/source-iotsitewise"
+  "@iot-app-kit/source-iottwinmaker"
+)
+
+# iot-app-kit CLI command initiator
+function iot-app-kit() {
+  # get the command (e.g., package, tarball, etc.) and then shift the argument stack
+  local cmd=$1; shift
+
+  # run the requested command and proxy the remaining/shifted arguments as a list (i.e., without the command)
+  if [ $cmd = package ]; then
+    iot-app-kit__package "$@"
+  elif [ $cmd = tarball ]; then
+    iot-app-kit__tarball "$@"
+  elif [ $cmd = --help ]; then
+    echo "
+${bold}USAGE${normal}
+  iot-app-kit <command> <subcommand> [flags]
+
+${bold}CORE COMMANDS${normal}
```

Comment:
Each command has specific arguments that should be listed in either this help menu or a command help menu.